registry.csv

```diff
@@ -0,0 +1,7 @@
+node_or_link,number,project
```

Comment:
Is this a registry for ADDED nodes or links or USED nodes or links in the project?  I think is is ADDED?  If so, suggest the following header:

`type,id,project_added` or similar